### PR TITLE
Microverse capstone project

### PR DESCRIPTION
**Pull request by Fombi-Favour**
In this activity, we:
- Added the index.html and about.html files containing designs
- Added the style.css and aboutStyle.css files for the styling of the two html files
- Added the index.js and expand.js for interactive styling and DOM manipulation
- Added image files for the design
- Added animations in some elements of the design

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,161 @@
...
+
+- **Beautiful mobile layouts**
+- **Site with nice styles**
+- **Great animation display**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link. Click here!](https://fombi-favour.github.io/Microverse-portfolio/)
```
Comment (usorfaitheloho) [inline README.md]: Please add a live link for this project to enable visitors have the right link to your site from your README file 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+          <a href="index.html"><img src="/assets/logo.png" alt="logo"></a>
+          <ul class="menu-list">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+          </ul>
+          <a href="" class="camp">GDC Campaign</a>
+        </div>
```
Comment (usorfaitheloho) [inline index.html:37]: You did well with your design, please implement the dark top navbar that contains the social icons and logout, it is always important to replicate a design given  😄

design given :
image


your design :
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      <div class="info">
+        <p>2022 Game Developers Conference. Some Rights Reserved.</p>
+        <p class="hidden">All contents on this website, unless stated otherwise, is at GDC in USA.
+        For detailed usage, function details and permission conditions, see <a href="https://gdconf.com/">https://gdconf.com/</a></p>
+      </div>
+    </footer>
+    <script src="index.js"></script>
+    <script src="expand.js"></script>
+  </body>
+</html>
```
Comment (usorfaitheloho) [inline index.html:137]: Please reduce the spacing between your footer items, the space between them is too much and looks quite different from what we have in the design which is a better UI 😸 

Design given :
image

Your design :
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        <div class="text">
+          <p>"Hello! Sharing world"</p>
+          <p class="title">GAME DEVELOPERS CONFERENCE SHOWCASE 2022</p>
+        </div>
+        <p class="sub-title"> A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in March, in USA.</p>
+        <div class="date-venue">
+          <p class="date">2022-03-21(MON) ~ 25(FRI)</p>
+          <p class="venue">@ Moscone Center San Francisco, CA USA</p>
+        </div>
+      </div>
```
Comment (usorfaitheloho) [inline index.html]: Kindly increase the font-size for the mobile view of your heading as it seems really small and not very legible compared to the design given 

design given :
image

Figma design :
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+      <a href="#" class="mobile-text">Join GDC 2022</a>
+    </section>
+    <!-- featured speakers section -->
+    <section class="speakers">
+      <h2 class="titles">Featured Speakers</h2>
+      <div class="speakers-list"></div>
+      <button type="button" class="btn-extend">
+        <span class="btn-text">More</span>
+        <img src="/assets/arrow_down.png" class="btn-pic" alt="down">
+      </button>
```
Comment (usorfaitheloho) [inline index.html:109]: Please for both the desktop and mobile version of the `featured speakers section` , you should add the top background image,as it should be looking at the design given. Please try to have a detailed perspective about your design to convince your client that you have an eyes for the job 😄 . Also you should take a closer look at your font for all paragrahs in your design, as it seems like you didnt use the right fon-type of `cococoose` and colors for the paragraphs, and they look different from your design

Design given :
image

Your design :
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+        <img src="/assets/partners/partner1.png" alt="partners">
+        <img src="/assets/partners/partner2.png" alt="partners">
+        <img src="/assets/partners/partner3.png" alt="partners">
+        <img src="/assets/partners/partner4.png" alt="partners">
+        <img src="/assets/partners/partner5.png" alt="partners">
+        <img src="/assets/partners/partner6.png" alt="partners">
+        <img src="/assets/partners/partner7.png" alt="partners">
+        <img src="/assets/partners/partner8.png" alt="partners">
+      </div>
+    </section>
```
Comment (usorfaitheloho) [inline index.html:124]: Please the mobile view of the homepage in the design given doesnt include the `partners` section, it is only included in the about page. Please can you make this correction 😄 

Design given :
image

Your design :
image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+        </div>
+        <div class="pic-col">
+          <img src="/assets/show2.jpg" alt="show1">
+          <div class="layer">
+            <h3>2020</h3>
+            <h4>GDC 2020 in San Francisco, CA USA</h4>
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (usorfaitheloho) [inline about.html:101]: Kindly make the following corrections for your `about` page :

- [x] Please include the background images as seen in the design given as yours looks really plain and doesn't really have a good user interface 😄 
- [ ] Also, kindly take note of the font sizes and coors used in the paragraphs in the main body of the about section as yours seems quite small and not legible for user with an eye sight defficiency, remeber to alsways consider different type of users while building your site
- [ ] The margin betwen the contact section of the about page and the next section after it is quite alot, can you kindly reduce it ?
- [ ] As i said earlier, you seem to have an inconsistency in your font-type. I think you should use `lato` for headings and `cocogoose` for paragraphs to enable a legible design
- [ ] kindly increase the width of the logo you attached in the `LOGO SECTION` of your about page and remeber to give it a border box property too 😄 
- [ ] If you take a close look at the pictures attachedin the `See the past game developers conference` section of your project, it should have a background color property, with the date and venue positioned on top of it, please implement these changes to get a better look

Your design : 

https://github.com/Fombi-Favour/microverse-capstone-1/assets/20537712/8ef5cc58-9081-4fe0-b7ca-20904988b40f


Behance design :

https://github.com/Fombi-Favour/microverse-capstone-1/assets/20537712/7b55fc79-7501-47a6-b7dc-91e3d00e5b15

File: about.html [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+    <footer class="diff">
+      <img src="/assets/frame-logo.png" class="frame" alt="logo">
+      <div class="info">
+        <p>2022 Game Developers Conference. Some Rights Reserved.</p>
+        <p class="hidden">All contents on this website, unless stated otherwise, is at GDC in USA.
+        For detailed usage, function details and permission conditions, see <a href="https://gdconf.com/">https://gdconf.com/</a></p>
+      </div>
+    </footer>
+    <script src="index.js"></script>
+  </body>
```
Comment (usorfaitheloho) [inline about.html:126]: Kindly use good semantics principles for your HTML code, remember that in the `hx` series, they come in the order of `h1,h2,h3 ...h6` meaning they should be followed in that hierarchy in your project

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Fombi-Favour  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @Fombi-Favour,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**